### PR TITLE
Edit site

### PR DESCRIPTION
**Pull request by IanDTrapp**

### CODE REVIEW

File: plugins/pencilblue/controllers/actions/admin/sites/edit_site_action.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
+module.exports = function(pb) {
```
Comment (btidwell) [inline plugins/pencilblue/controllers/actions/admin/sites/edit_site_action.js]: Can we name this function to "EditSiteActionModule"?  So use "module.exports = function EditSiteActionModule(pb)"

File: plugins/pencilblue/controllers/admin/sites/edit_sites.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
+module.exports = function(pb) {
```
Comment (btidwell) [inline plugins/pencilblue/controllers/admin/sites/edit_sites.js]: Same thing here.  "function EditSitesModule(pb)"

**Comment by btidwell** [comment]
For edit_sites.js, I don't think you need this extra controller as it's functionality could be incorporated into site_form.js.  The rest of the pencilblue admin items uses one controller for it's form pages regardless if it's a new item or an edit of an existing item.  However, they do use two separate action controllers for edit and new.

**Comment by IanDTrapp** [comment]
@btidwell Changes made, open for review once again

**Pull request merged**